src/lib.rs

```diff
@@ -1116,6 +1142,27 @@ impl<'a, 'input: 'a> Node<'a, 'input> {
     pub fn range(&self) -> Range {
         self.d.range.clone()
     }
+
+    /// Returns node's NodeId
+    /// 
+    /// # Examples
+    /// 
+    /// ```
+    /// let doc = roxmltree::Document::parse("\
+    /// <p>
+    ///     text
+    /// </p>
+    /// ").unwrap();
+    /// 
+    /// use roxmltree::NodeId;
+    /// (0..doc.descendants().count()).for_each(|n| {
```

Comment:
This is a very complicated example. We should simplify it or remove it completely.

Reply:
I will remove it, since this is just a getter-method now, essentially.